src/main/kotlin/com/grepp/nbe1_3_team04/reservation/service/response/ReservationInfoDetailsResponse.kt

```diff
@@ -0,0 +1,35 @@
+package com.grepp.nbe1_3_team04.reservation.service.response
+
+import com.grepp.nbe1_3_team04.reservation.domain.*
+import java.time.LocalDateTime
+
+
+data class ReservationInfoDetailsResponse(
+    val courtName: String,
+    val matchTeamName: String,
+    val matchDate: LocalDateTime,
+    val participants: List<ParticipantInfoResponse>,
+    val gender: ParticipantGender,
+    val status: ReservationStatus
+) {
+    companion object {
+        fun of(
+            reservation: Reservation,
+            participants: List<Participant>,
+            matchTeamName: String
+        ): ReservationInfoDetailsResponse {
+            val participantResponses = participants.map { participant ->
+                ParticipantInfoResponse.from(participant)
+            }
```

Comment:
```suggestion
            val participantResponses = participants.map(ParticipantInfoResponse::from)
```
람다식 말고 Kotlin의 매서드 참조 방식으로도 작성할 수 있습니다! 그냥 알아만 두시면 좋을 것 같습니다.

Reply:
더 좋은 방법이 있었군요!! 감사합니다😄